boot/boot.js

```diff
@@ -287,6 +287,42 @@ $tw.utils.decodeURIComponentSafe = function(s) {
 	return v;
 };
 
+
+/*
+Convert a URI List Component encoded string (with `+` as an allowed replacement 
+for `+`) to a string
+*/
+$tw.utils.decodeTWURIList = function(s) {
+	return $tw.utils.decodeURIComponentSafe(
+		s.split('&')
+		.map(function(s) {return s.replaceAll('+', ' ')})
+		.map(function(s) {return s.includes(' ') ? '[[' + s + ']]' : s})
```

Comment:
`array.prototype.includes()` is ES2016 and `replaceAll()` is ES2021 ... TW atm uses ES5 only
see: http://kangax.github.io/compat-table/es2016plus/

Reply:
Fixed in 3249069e6a84610c29ac74a92fe780d5423e7077